tmk_core/protocol/report.h

```diff
@@ -199,6 +199,12 @@ typedef int16_t mouse_xy_report_t;
 typedef int8_t mouse_xy_report_t;
 #endif
 
+#ifdef WHEEL_EXTENDED_REPORT
+typedef int16_t mouse_hv_report_t;
+#else
+typedef int8_t mouse_hv_report_t;
+#endif
+
```

Comment:
Has been included in [a separate PR](https://github.com/qmk/qmk_firmware/pull/24422).